public/docs/getting_started/client.md

```diff
@@ -4,11 +4,24 @@ There are many Ethereum clients to choose from. We recommend different clients d
 
 ## When developing
 
-### Truffle Develop
+### Ganache
+
+We recommend [Ganache](/ganache), a one-click GUI blockchain you can use for development. It is designed to be interoperable with Truffle. It processes transactions instantly instead of waiting for the default block time -- so you can test that your code works quickly -- and it tells you immediately when your smart contracts run into errors. It also makes a great client for automated testing, and Truffle knows how to use its special features to speed up test runtime by almost 90% compared to other clients.
```

Comment:
I want to edit the sell here to differentiate Ganache from our other chains (Ganache is all about seeing, personalization, and speed -- and likely more. We'll tighten this up as Ganache matures):  

> We recommend [Ganache](/ganache), a personal blockchain for Ethereum development that runs on your desktop. Part of the Truffle suite, Ganache simplifies dapp development by placing your contracts and transactions front and center. Using Ganache you can quickly see how your application affects the blockchain, and introspect details like your accounts, balances, contract creations and gas costs. You can also fine tune Ganache's advanced mining controls to better suit your needs. Ganache is available for Windows, Mac and Linux, and you can [download it now](/ganache).